Manual.md

```diff
@@ -113,56 +70,317 @@ Each container directory must be on a btrfs filesystem, and is created as a btrf
 <dt>wal_archive</dt>
   <dd>WAL segements retrieved by streaming replication from the upstream DB.</dd>
 <dt>standby</dt>
-  <dd>A btrfs subvolume containing a PostgreSQL data dir for a warm-standby instance. Snapshots are created from this directory.</dd>
+  <dd>A btrfs subvolume containing a PostgreSQL data dir for a warm-standby instance. Snapshots are created from this directory. It uses an embedded socket dir and turns off TCP so as not to interfere with any other PostgreSQL instances on localhost.</dd>
 <dt>snapshots</dt>
-  <dd>Each subdirectory here is a btrfs read-only snapshot of the standby directory, with a ISO 8601 timestamp as the name (always UTC).</dd>
+  <dd>Each subdirectory here is a btrfs read-only snapshot of the standby directory, with a ISO 8601 timestamp as the name (always UTC). Note snapshots lack pg_xlog and pg_log directories because these are symlinks.</dd>
 <dt>clones</dt>
   <dd>Each subdirectory is a write-able btrfs subvolume created by snapshotting a snapshot yet again. The names of clones is user-determined.</dd>
 </dl>
 
-Each running container consists of 2 process groups, one a pg_receivexlog process maintaining the wal_archive, the other one a minimal PostgreSQL instance, like this:
+### Running Services
+
+Each running container consists of 2 process groups, one a pg_receivexlog process maintaining the wal_archive and a minimal PostgreSQL instance, like this:
 
 ```
 $ ps xf
- 4737 ?        S      0:01 pg_receivexlog -D /mnt/backup/<NAME>/wal_archive \
+ 4737 ?        S      0:01 pg_receivexlog -D /srv/backup/<NAME>/wal_archive \
```

Comment:
It's a public repository, and we don't need to promote local violations of the FHS.